acinclude.m4

```diff
@@ -249,8 +249,10 @@ AC_DEFUN([MK_CHECK_CA_BUNDLE], [
     AC_DEFINE_UNQUOTED(MK_CA_BUNDLE, "$ca", [Location of default ca bundle])
     AC_SUBST(MK_CA_BUNDLE)
     AC_MSG_RESULT([$ca])
-  else
+  elif test "x$cross_compiling" == "xyes"; then
     AC_MSG_RESULT([no])
```

Comment:
ok, thanks. The actual output while cross-compiling is:

    checking CA bundle path... skipped (cross compiling)
    configure: WARNING: skipped the ca-bundle detection when cross-compiling

We need the second output because otherwise when running configure with -q (as in the build ios script) we are not able to see that the ca-bundle search has been skipped.